src/node.cc

```diff
@@ -1155,7 +1154,6 @@ bool ShouldAbortOnUncaughtException(Isolate* isolate) {
   return isEmittingTopLevelDomainError || !DomainsStackHasErrorHandler(env);
```

Comment:
I would like to understand why the returned `caught` boolean in `process._fatalException` is not enough to determine if we should abort on `uncaughtException`.